src/db/entities/coin.ts

```diff
@@ -0,0 +1,10 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity()
+export class Coin {
+  @PrimaryGeneratedColumn()
+  id!: number;
+
+  @Column({ type: 'varchar' })
+  name!: string;
```

Comment:
We need more data about the coin!
Please go ahead and send me (or comment here) all the data that is returned by CoinGeko API and I'll tell you which one to put.

Reply:
It returns a list of the coin's respective price, market data, historical data, and metadata (image, descriptions, links, followers, token supply info, etc.) for example ID, symbol, name, current price for all other coins, scores, price changes and the percentages, last updated, etc. there are too many data.

Reply:
For coin entity, try having:
- id
- symbol
- name
- total_supply

Reply:
done.